### DISCUSSION TITLE
dpgen run error in training to task04

### DISCUSSION THREAD

**Discussion by LZH-1996** [deepmodeling/dpgen#955] [Q&A] [answered]
According to the process, when 00.train runs to get the initial model, then use lammps to explore the space (01.model_devi), but there will always be a failure to submit the task, and I checked the files and logs more than once, but I did not find any specific reasons for this fault. and attach the inputfile，logs
![dp](https://user-images.githubusercontent.com/46983771/190574687-9171e627-11e1-4b5b-b532-d846a700e31c.PNG)
![dp2](https://user-images.githubusercontent.com/46983771/190574779-8dc18dac-0e73-418a-bda4-f4dbe10e15dc.PNG)
![dp3](https://user-images.githubusercontent.com/46983771/190574832-431d7d9d-e277-48e3-b233-52ece48b64ae.PNG)

**Comment by HuangJiameng** [comment, marked as answer]
Could you check and provide the log file on `remote_root`, which is mentioned in this message. 
![image](https://user-images.githubusercontent.com/105633685/190593796-89b6376e-075b-4422-9b95-7d217144b0d5.png)

**Reply by LZH-1996** [reply]
thank you, i checked the dir and found  wrongs  may caused for lammps setting, now its run again, i will continue to follow the results.

**Reply by chb-deep** [reply]
Hello, I also reported an error at task 04, and the model_devi process could not be carried out, according to the error message, it was also in the remote_root directory, but I don't know how to modify it, how did you solve this problem?
